app/config/errors.php

```diff
@@ -581,7 +581,7 @@
     ],
     Exception::INDEX_ALREADY_EXISTS => [
         'name' => Exception::INDEX_ALREADY_EXISTS,
-        'description' => 'Index with the requested ID already exists. Try again with a different ID or use "unique()" to generate a unique ID.',
+        'description' => 'Index with the requested ID already exists. Try again with a different ID or use ID.unique() to generate a unique ID.',
```

Comment:
We also call it keys for indexes...Also, I don't think you can use unique for index keys.